spec/receipt_spec.rb

```diff
@@ -36,34 +36,33 @@
               'is_trial_period' => 'false'
             }
           ],
-          'original_json_response' => {
-            'pending_renewal_info' => [
-              {
-                'auto_renew_product_id' => 'com.foo.product1',
-                'original_transaction_id' => '37xxxxxxxxx89',
-                'product_id' => 'com.foo.product1',
-                'auto_renew_status' => '0',
-                'is_in_billing_retry_period' => '0',
-                'expiration_intent' => '1'
-              }
-            ]
+        },
+        'pending_renewal_info' => [
+          {
+            'auto_renew_product_id' => 'com.foo.product1',
+            'original_transaction_id' => '37xxxxxxxxx89',
+            'product_id' => 'com.foo.product1',
+            'auto_renew_status' => '0',
+            'is_in_billing_retry_period' => '0',
+            'expiration_intent' => '1'
           }
-        }
+        ]
       }
     end
 
-    subject { Venice::Receipt.new(response['receipt']) }
+    subject { Venice::Receipt.new(attributes: response['receipt']) }
 
-    its(:bundle_id) { 'com.foo.bar' }
-    its(:application_version) { '2' }
+    its(:bundle_id) { should eql 'com.foo.bar' }
```

Comment:
In the `its` statements from line 57 to 65, the ones that did not use `should` were actually not asserting anything. (False positives.)